### DISCUSSION TITLE
Porting to Godot 4?

### DISCUSSION THREAD

**Discussion by Overvault-64** [Shin-NiL/Godot-Android-Admob-Plugin#210] [General]
Hello there, I'm trying to porting this plugin to Godot 4 but I'm having issues opening the project.

The current `.aar` works fine but I wanted to update the dependencies (including the [Godot one](https://mvnrepository.com/artifact/io.github.m4gr3d/godot/4.0.0.beta3)) to the latest versions.

So I wanted to know if @Shin-NiL are you going to port the plugin to Godot 4 eventually?

If yes, here's my porting of `admob.gd`.
[admob.zip](https://github.com/Shin-NiL/Godot-Android-Admob-Plugin/files/10663963/admob.zip)

I added optional arguments 
to load methods to allow passing ids at runtime.

**Comment by Theo34** [comment]
Good Day @Shin-NiL 

I also would like to find out if you are porting this plugin to Godot 4 ?

Thank you for your work
